layouts/_default/list.html

```diff
@@ -0,0 +1,26 @@
+{{ define "main" }}
+
+<div class="flex-l mt2 mw8 center">
+  <article class="center cf pv5 ph3 ph4-ns mw7">
+    <header>
+      <h1 class="f1">
+        {{ .Title }}
+      </h1>
+    </header>
+    <div class="nested-copy-line-height lh-copy f4 nested-links nested-img mid-gray">
+      {{ .Content }}
+    </div>
+  </article>
+</div>
+
+  <article class="pa3 pa4-ns nested-copy-line-height nested-img">
+    <section class="flex-ns flex-wrap justify-around mt5">
+      {{ range .Paginator.Pages }}
+        <div class="relative w-100 w-30-l mb4 bg-white">
+          {{- .Render "summary" -}}
```

Comment:
```suggestion
          {{- partial "summary.html" . -}}
```

Reply:
I just changed this to the `.Render` thing because of @verwoerd s suggestion to avoid the deprication warning.
I think the problem is that we are not all on the same version of the theme.

In #15 @verwoerd updated the theme, so if that is merged first, I think it should fix it maybe.

Reply:
For clarity sake, I have seperated the theme update in a seperate pr #17 so its not tied to the archive LFS discussion.

Reply:
I verified that this indeed fixes it, by merging the two branches locally, thanks! :smile: